Dockerfile

```diff
@@ -11,7 +11,7 @@ ENV HIVE_VERSION=${HIVE_VERSION}
 
 ENV JAVA_VERSION 1.8.0
 ENV SHUNTING_YARD_VERSION 3.0.0
-ENV CIRCUS_TRAIN_VERSION 13.2.1
+ENV CIRCUS_TRAIN_VERSION 14.1.0
```

Comment:
We're sure that shunting yard won't be affected by the config changes for Housekeeping that resulted in the major version bump of CT from 13 to 14? Or do we need a new version of Shunting Yard to go along with this (i.e. one using the same version of CT).

Reply:
Yep Shunting Yard will be changing to check for the Beekeeper param in the Apiary event in order to build the CT config. I think I've opened this a little prematurely.

Reply:
OK, so why don't we change and release shunting yard and then come back here and update both the SY and CT versions at the same time?

Reply:
Yep thanks Adrian, I'm going to close this in the meantime